### ISSUE TITLE
[RAML to Code] Support other versions of Spring MVC

### ISSUE THREAD

**Issue by kurtpa** [phoenixnap/springmvc-raml-plugin#9]
Add a toggle and support for legacy versions of Spring MVC in the generation of Spring server endpoints

**Comment by kurtpa** [comment]
From Erreobi:

```java
		SchemaMapper mapper = new SchemaMapper(new RuleFactory(config, new Jackson1Annotator(), schemaStore),
				new SchemaGenerator());
```

you can choose between ``Jackson1Annotator`` or ``Jackson2Annotator``.

**Comment by kurtpa** [comment]
Ready to be released with 0.4.1.
